go/slack/slack_test.go

```diff
@@ -0,0 +1,24 @@
+package slack
+
+import (
+	"github.com/stretchr/testify/require"
```

Comment:
Based on Fatih's suggestions, I've started using [quicktest](https://github.com/frankban/quicktest) in #70 and #83. Unless you want to continue with Testify, I think we should continue using QuickTest over Testify throughout the whole codebase.